### ISSUE TITLE
Added voucher codes not showing in order adjustments

### ISSUE THREAD

**Issue by jmauzyk** [verbb/gift-voucher#65]
**Description**

Voucher codes that have been added to an order are not affecting order total and do not display in the order adjustments on CP order edit pages. Was this behavior intentionally changed, or is this potentially an issue with the most recent versions of Craft Commerce?

**Steps to reproduce**
1. From CP order edit page, add a voucher code.
2. Code is successfully added to the order and the page refreshes, but no voucher code is displayed in adjustments and order total remains the same.

**Additional info**
- Plugin version: 2.5.9
- Craft version: Pro 3.6.12.1
- Multi-site: No

**Comment by engram-design** [comment]
Can't seem to replicate this immediately on Commerce 3.3.0.1
![image](https://user-images.githubusercontent.com/1221575/116637140-73f47d00-a9a6-11eb-83ef-86c612ad7ec4.png)

**Comment by jmauzyk** [comment]
@engram-design I dug into this a bit more and was able to determine that this has to do with the new options for the Shipping Rule _Order Total Value_ condition. When `minMaxTotalType` is set to _Value of products with applied discounts_, the voucher code is saved to the session, but the corresponding adjustment does not get saved to the order. The logic for this new setting is located starting at line 332 of `craft\commerce\models\ShippingRule`.

**Comment by engram-design** [comment]
Ah, you're right on the money with this one. How bizarre that a shipping rule setting would affect a discount from a gift voucher like this, but I can see from the code how this and potentially other things might catch this issue.

Fixed for the next release. To get the fix early, change your `verbb/gift-voucher` requirement in `composer.json` to:

```
"require": {
  "verbb/gift-voucher": "dev-craft-3 as 2.5.9",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.5.10](https://github.com/verbb/gift-voucher/releases/tag/2.5.10)

**Comment by engram-design** [comment]
FYI, see my comment https://github.com/verbb/gift-voucher/issues/69#issuecomment-893344453 about this.
